ui_src/src/components/modal/index.js

```diff
@@ -27,7 +27,7 @@ import { Modal } from 'antd';
 import Button from '../button';
 
 const TransitionsModal = (props) => {
-    const { height, width, rBtnText, lBtnText, rBtnDisabled, lBtnDisabled, header, isLoading, open = false, displayButtons = true } = props;
+    const { height, width, rBtnText, lBtnText, rBtnDisabled, lBtnDisabled, header, isLoading, headerImage, subTitle, open = false, displayButtons = true } = props;
```

Comment:
You don't need a headerImage to display the image, you can send <Img /> as a header prop, and that will work.

Reply:
Fixed